core/src/main/java/org/bitcoinj/script/ScriptError.java

```diff
@@ -1,3 +1,19 @@
+/*
+ * Copyright 2013 Google Inc.
```

Comment:
oh... I've copy/pasted without reading.
I've seen that also Script.java contains your copyright. Could I add my name too?